README.md

```diff
@@ -1,58 +1,17 @@
-### VSTest
```

Comment:
I think this is the main readme right? Should we put the new one inside playground folder?